src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1571,10 +1592,19 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
-
-    // TODO: Add desired media content for upload. For more information, see:
+    // TODO: Use `media` to add desired media content for upload. For more information, see:
```

Comment:
Oh, I see. The nodsjs changes in #538 supersede these, correct?

But the structure for both upload and download is now this, correct?:
```
// TODO: blah blah
// For more information, see xxx
someCode = goes.Here
```